met-api/src/met_api/models/widget_item.py

```diff
@@ -0,0 +1,62 @@
+"""WidgetItem model class.
+
+Manages the widget_item
+"""
+from __future__ import annotations
+from datetime import datetime
+
+from sqlalchemy.sql.schema import ForeignKey
+
+
+from .db import db
+
+
+class WidgetItem(db.Model):  # pylint: disable=too-few-public-methods
+    """Definition of the WidgetItem entity."""
+
+    __tablename__ = 'widget_item'
+    __table_args__ = (
+        db.UniqueConstraint('widget_data_id', 'widget_id', name='unique_widget_data'),
+    )
+
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    widget_data_id = db.Column(db.Integer, nullable=False)
```

Comment:
Please add some good comments explaining our reasoning behind the widget_data_id and how it helps our scenarios.. We will eventually forget whats this widget_data_id is

Reply:
Added comment